Formula/signal-cli.rb

```diff
@@ -0,0 +1,17 @@
+class SignalCli < Formula
+  desc "CLI and dbus interface for WhisperSystems/libsignal-service-java"
+  homepage "https://github.com/AsamK/signal-cli"
+  url "https://github.com/AsamK/signal-cli/releases/download/v0.6.1/signal-cli-0.6.1.tar.gz"
+  sha256 "550b389a25089ef2251d97997e06d57c70e9d4023684ff8682afef23ed2f1822"
+  depends_on :java => "1.7+"
+
+  def install
+    # pattern taken from activemq
+    libexec.install Dir["lib", "bin"]
+    (bin/"signal-cli").write_env_script libexec/"bin/signal-cli", Language::Java.java_home_env("1.7+")
+  end
+
+  test do
+    system "#{bin}/signal-cli", "--version"
```

Comment:
Cab you create a test that test the functionality of the software rather than printing the version number ?

Reply:
It's a little difficult because it's a secure messaging client. 

If `--version` works, it guarantees class loading is working and the install is good.

The `link` command (and the app in general) requires a specific patch level of java or a java setting enabled, so it's not a pure test that the installation worked.

I've pushed a fix that continues to check version (but also checks the version is correct) and checks that the `link` command works.

Reply:
Not sure if you're happy with the link test, but will mark as resolved.

Reply:
Maybe we can try to read messages like `signal-cli -u homebrew receive` and assert message that it's not authenticated or something.